snap/snapcraft.yaml

```diff
@@ -82,7 +82,7 @@ parts:
       - python3-pip
     build-attributes: [enable-patchelf]
     stage:
-      - '*'
+      - "*"
```

Comment:
Nitpick: we should agree on single vs double quotes in yaml files at some points - I have custom linter settings for several projects by now :D

Reply:
I think thats my auto save making that change sorry :(